our-brothers-web/src/app/agenda/page/agenda-page.component.html

```diff
@@ -115,7 +115,12 @@ <h2 class="medium-heading" data-aos="fade-up" data-aos-duration="1000">
         <ng-container *ngFor="let quote of page.page" data-aos="fade-up" data-aos-duration="1000">
           <div class="quote">
             <img class="apostrophy" src="../../../assets/img/apostrophy.png" />
-            <div class="site-text quote-text">{{ quote.excerpt }}</div>
+            <a
+              class="site-text quote-text"
+              target="_blank"
+              [href]="'../../../assets/texts/' + quote.writerId + '.pdf'"
```

Comment:
Can we remove file type?

Reply:
I checked it out and it's won't work without file extension

Reply:
Make sure you use the file extension inside the `quote.writerId`. Also, consider to rename it.